ontoconv/pipelines.py

```diff
@@ -164,16 +165,25 @@ def generate_ontoflow_pipeline(  # pylint: disable=too-many-branches,too-many-lo
     nodes,
     save_final_output=False,
     recognised_keys: "Optional[Union[dict, str]]" = "basic",
+    target_ts: "Optional[Triplestore]" = None,
 ) -> dict:
     """Return a declarative ExecFlow pipeline as a dict.
 
     Arguments:
-        ts: Tripper triplestore documenting data sources and sinks.
+        ts: Tripper triplestore documenting data sources and model
+            inputs and outputs.
         resources: A dict referring to OTEAPI partial pipelines for
             a set of data sources and sinks.
             See the example below for the expected structure.
         client_iri: IRI of OTELib client to use.
-
+        recognised_keys: Dict that maps keys to IRIs (in an ontology).
+            These are used to detect keys in the knowledge base.
+            E.g. 'downloadUrl' is recognised in the kb as such a key.
```

Comment:
Note that OTEAPI spells this wrong. It should be `dcat:downloadURL`
Must be changed when/if we start using `tripper.dataset`

Reply:
Should we add that both downloadUrl and downloadURL are recognised?